#!/usr/bin/env python
PACKAGE = "controls"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("xB", double_t, 0, "Tryphon : x", 0,  -3, 3)
gen.add("yB", double_t, 0, "Tryphon : y", 0,  -3, 3)
gen.add("zB", double_t, 0, "Tryphon : z",    2.5, 2, 4)
gen.add("yawB", double_t, 0, "Tryphon : yaw",  0 , -3.14,  3.14)

gen.add("xE", double_t, 0, "Tryphon : x", 0,  -3, 3)
gen.add("yE", double_t, 0, "Tryphon : y", 0,  -3, 3)
gen.add("zE", double_t, 0, "Tryphon : z",    2.5, 2, 4)
gen.add("yawE", double_t, 0, "Tryphon : yaw",  0 , -3.14,  3.14)

gen.add("temps", double_t, 0, "Tryphon : time", 10,  0.1, 100)

gen.add("path", bool_t, 0, "Tryphon : path",  False)



exit(gen.generate(PACKAGE, "controls", "trajectory"))
